### DISCUSSION TITLE
Title "# Sign" Fails to Show Up

### DISCUSSION THREAD

**Discussion by yenchenlin** [pradyunsg/furo#494] [Questions] [answered]
Hi thanks for the nice theme. I followed the instruction to `pip install furo` and change the theme of my documentation but faced this issue. Any suggestion?

![Image from iOS](https://user-images.githubusercontent.com/7057863/182212847-476a6c81-1681-426f-9c24-8072f9f0ec03.jpg)

**Comment by pradyunsg** [comment]
You likely have html_permalinks in your conf.py

**Comment by yenchenlin** [comment, marked as answer]
I checked but didn't find `html_permalinks`. [Here](http://codepad.org/qP5tlYe3) is my `conf.py`.

Thanks for the timely reply BTW.

**Reply by yenchenlin** [reply]
I found the reason. It's because I have 'sphinx_rtd_theme' in my extensions.

**Reply by pradyunsg** [reply]
Yea, sphinx-rtd-theme is setting that for you. :)

**Reply by yenchenlin** [reply]
Thank you for your help!
